### ISSUE TITLE
Pole Density Function

### ISSUE THREAD

**Issue by maclariz** [pyxem/orix#484]
Just had some fun with pole density function today, but I found an issue with documentation and I have a question.

First the issue:
This page has plotting into an existing figure:
https://orix.readthedocs.io/en/stable/tutorials/pole_density_function.html
But this page doesn't even mention a "figure" argument you can call:
https://orix.readthedocs.io/en/stable/reference/generated/orix.measure.pole_density_function.html

Secondly, the question:
Can I use this function to plot within an existing defined axis within a multiple axis figure (e.g. created with plt.subplots)?

**Comment by harripj** [comment]
@maclariz you're right. The [`measure.pole_density_function`](https://orix.readthedocs.io/en/stable/reference/generated/orix.measure.pole_density_function.html#orix.measure.pole_density_function) function performs the computation and returns the histogram data and edges, and is useful for accessing the data or performing further processing on the histogram data.

There is a similarly named function (perhaps confusingly) [`StereographicPlot.pole_density_function`](https://orix.readthedocs.io/en/stable/reference/generated/orix.plot.StereographicPlot.pole_density_function.html#orix.plot.StereographicPlot.pole_density_function) which is essentially a convenience function around `measure.pole_density_function` mentioned above, and can be used to quickly produce a plot. You can see this in the code [here](https://github.com/pyxem/orix/blob/7a42ea2f6c4ae2fd80be3bf89ee31c652e473fe7/orix/plot/stereographic_plot.py#L314). (There is also a density plot for the [inverse pole figure](https://orix.readthedocs.io/en/stable/reference/generated/orix.plot.InversePoleFigurePlot.pole_density_function.html#orix.plot.InversePoleFigurePlot.pole_density_function)).

To answer your second point, yes absolutely. There are a couple examples in the tutorial you linked above. You just need to define the subplot projection as `stereographic` to use the [`StereographicPlot`](https://orix.readthedocs.io/en/stable/reference/generated/orix.plot.StereographicPlot.html#orix.plot.StereographicPlot) functions. 

Here is an example with an independent plot (note that the `orix.plot` import is necessary to register the projection within Matplotlib):
```python
import orix.plot
from orix.vector import Vector3d

import numpy as np
import matplotlib.pyplot as plt

v = Vector3d(np.random.randn(1_000, 3))

fig = plt.figure(figsize=(10, 5))
ax1 = fig.add_subplot(121, projection="stereographic")
ax2 = fig.add_subplot(122)

ax1.pole_density_function(v)
x = np.arange(10)
ax2.plot(x, x**2)
```
![image](https://github.com/pyxem/orix/assets/16853829/d3655d41-a59a-4dae-9d47-8df7d7359838)

**Comment by hakonanes** [comment]
This should be added to the examples gallery!

**Comment by maclariz** [comment]
Thanks.  This is helpful.  Will use this.  Sorry for late reply, was travelling on a different project.
